metrics_api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -60,12 +62,14 @@ def create_instrument(kind, name, unit, description, callback)
         raise InstrumentNameError if name.empty?
         raise InstrumentNameError unless NAME_REGEX.match?(name)
         raise InstrumentUnitError if unit && (!unit.ascii_only? || unit.size > 63)
-        raise InstrumentDescriptionError if description && (description.size > 1023 || !utf8mb3_encoding?(description))
+        raise InstrumentDescriptionError if description && (description.size > 1023 || !utf8mb3_encoding?(description.dup))
```

Comment:
I'll have to re-review this, there was some wording in the API specifically around it being a failure if it was not utf8mb3.  That may have changed since adding this.

Reply:
Ah, that would make sense. I may have missed that bit.

Reply:
I read [this](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-description) as "you _must_ support utf8m3" but don't see anything about blowing up if desc is not utfmb3.

Reply:
To be fair, I'm okay with being a little strict and then addressing the case later on when someone uses emoji in their metrics names or something. 😄